src/functional/resources/repo-config.csv

```diff
@@ -1,4 +1,4 @@
-Repository's Location,Branch,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Beta.git,master,docs**,yes
-https://github.com/reposense/testrepo-Charlie.git,master,,
-https://github.com/reposense/testrepo-Delta.git,master,,
+Repository's Location,Branch,Ignore Glob List,Ignore standalone config,Ignore Commits List
+https://github.com/reposense/testrepo-Beta.git,master,docs**,yes,
+https://github.com/reposense/testrepo-Charlie.git,master,,,
+https://github.com/reposense/testrepo-Delta.git,master,,,
```

Comment:
we need to include it in functional test as well. Let's use charlie (in this config) and delta (standalone).

Reply:
will add some 'large' commits in the functional tests then